hooks/post-checkout

```diff
@@ -1,17 +1,69 @@
 #!/bin/bash
 
-default_version="0.29.2"
-version="${BUILDKITE_PLUGIN_TRIVY_VERSION:-$default_version}"
-image="aquasec/trivy:${version}"
+DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+
+# shellcheck source=lib/trivy-env.sh
+. "$DIR/../lib/trivy-env.sh"
```

Comment:
We spoke about `source`ing stuff over slack. We decided that we would be better off without it.

Reply:
Done.